Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -319,13 +321,13 @@ internal final class __DataStorage : @unchecked Sendable {
 
     @inlinable // This is @inlinable despite escaping the __DataStorage boundary layer because it is trivially computed.
     func get(_ index: Int) -> UInt8 {
```

Comment:
Should we validate this index? It would be technically source-breaking, but we never promised not to validate it.